docs/references/amp-constants-example.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2022 Alteryx, Inc. All rights reserved.
+#
+# Licensed under the ALTERYX SDK AND API LICENSE AGREEMENT;
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.alteryx.com/alteryx-sdk-and-api-license-agreement
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Example input tool."""
+from collections import namedtuple
+
+from ayx_python_sdk.core import PluginV2
+from ayx_python_sdk.providers.amp_provider.amp_provider_v2 import AMPProviderV2
+
+from pyarrow import Table
+
+
+class ConstantsExample(PluginV2):
+    """Concrete implementation of an AyxPlugin."""
```

Comment:
TW review: Update to title case, remove period

Concrete Implementation of an AyxPlugin

Reply:
Derek said we incorporated the change.